src/include/network/lan/udev.rb

```diff
@@ -45,6 +45,23 @@ def update_udev_rule_key(rule, key, value)
       rule
     end
 
+    # Returns a value of the particular key in the rule
+    #
+    # @return [string] value corresponding to the key or empty string
+    def udev_key_value(rule, key)
+      raise ArgumentError if rule.nil?
+
+      rule.each do |tuple|
```

Comment:
well it is one rule. But in present code it is represented as an array ... e.g. ```KERNELS=pciid, NAME="eth0"``` is currently represented as ```["KERNELS=pciid", NAME="eth0"]````